share/poly.go

```diff
@@ -199,18 +199,38 @@ func RecoverSecret(g kyber.Group, shares []*PriShare, t, n int) (kyber.Scalar, e
 	return acc, nil
 }
 
-func xScalar(g kyber.Group, shares []*PriShare, t, n int) map[int]kyber.Scalar {
+// xScalar returns the list of (x_i, y_i) pairs indexed. The first map returned
+// is the list of x_i and the second map is the list of y_i, both indexed in
+// their respective map at index i.
+func xScalar(g kyber.Group, shares []*PriShare, t, n int) (map[int]kyber.Scalar, map[int]kyber.Scalar) {
+	// we are sorting first the shares since the shares may be unrelated for
```

Comment:
What do you mean by "unrelated"? Is that the case when we construct a new polynomial via Lagrange interpolation in the DKG re-sharing case?

Reply:
Yes exactly. Uncorrelated may be a better word ?